website/docs/r/schema_template_bd.html.markdown

```diff
@@ -29,12 +29,13 @@ resource "mso_schema_template_bd" "bridge_domain" {
 
 * `schema_id` - (Required) SchemaID under which you want to deploy Bridge Domain.
 * `template_name` - (Required) Template where Bridge Domain to be created.
-* `name` - (Required) Name of Bridge Domain.
+* `name` - (Required) Name of the Bridge Domain.
 * `display_name` - (Required) Display Name of the Bridge Domain on the MSO UI.
-* `vrf_name` - (Required) Name of VRF to attach with Bridge Domain. VRF must exist.
-* `vrf_schema_id` - (Optional) SchemaID of VRF. schema_id of Bridge Domain will be used if not provided. Should use this parameter when VRF is in different schema than BD.
-* `vrf_template_name` - (Optional) Template Name of VRF. template_name of Bridge Domain will be used if not provided. Should use this parameter when VRF is in different schema than BD.
-* `layer2_unknown_unicast` - (Optional) Type of layer 2 unknown unicast. Allowed values are `flood` and `proxy`. Default to `flood`.
+* `description` - (Optional) Description of the Bridge Domain on the MSO UI.
+* `vrf_name` - (Required) Name of the VRF to attach with Bridge Domain. VRF must exist.
+* `vrf_schema_id` - (Optional) SchemaID of the VRF. schema_id of Bridge Domain will be used if not provided. Should use this parameter when VRF is in different schema than BD.
+* `vrf_template_name` - (Optional) Template Name of the VRF. template_name of Bridge Domain will be used if not provided. Should use this parameter when VRF is in different schema than BD.
```

Comment:
Template Name of the VRF. template_name of the Bridge Domain will be used if not provided. This parameter is required when the VRF is in a schema which is different from the schema of the BD.